### ISSUE TITLE
Add rule for incorrectly marked scenario outline

### ISSUE THREAD

**Issue by vsiakka** [vsiakka/gherkin-lint#43]
eg
```
Scenario: 
 Given I do this

Examples:
 |a| 
```

This currently crashes the parser, should make it print a nice rule message instead.
